src/lib/services-manager/dialogs/services_manager.rb

```diff
@@ -291,6 +281,15 @@ def handler_name(input)
         handler
       end
 
+      # Handler for help event (help button is used)
+      #
+      # A popup with help is shown
+      def help_handler
+        self.finish = false
```

Comment:
Just out of curiosity, why this is needed here?

Reply:
Only to ensure that the dialog will not be closed after handling this event.